TTS/tts/datasets/dataset.py

```diff
@@ -201,7 +209,7 @@ def get_phonemes(self, idx, text):
     def get_f0(self, idx):
         out_dict = self.f0_dataset[idx]
         item = self.samples[idx]
-        assert item["audio_file"] == out_dict["audio_file"]
+        assert item["audio_unique_name"] == out_dict["audio_unique_name"]
```

Comment:
Does it make sense to check if ```audio_unique_name``` is really unique in __init__?

Reply:
It was already in the code I just replaced the "audio_file" with "audio_unique_name". Both are equivalent, the "audio_unique_name" will be unique for all the audios and the "audio_file" suppose to be too.